icons/circle/48/mailspring.svg

```diff
@@ -0,0 +1,20 @@
+<svg xmlns="http://www.w3.org/2000/svg" version="1.1" xmlns:xlink="http://www.w3.org/1999/xlink" viewBox="0 0 48 48">
+ <defs>
+  <linearGradient id="linearGradient862" y2="24" gradientUnits="userSpaceOnUse" x2="47" gradientTransform="matrix(0 -1 1 0 -.0000015 48)" y1="24" x1="1">
```

Comment:
* Instead of calling it `linearGradient862` you could name it `bg` (or even `b`)
* `gradientTransform` is unnecessary. Please look at the _hacknet icon_ for an example